### DISCUSSION TITLE
6월 2주차 회의

### DISCUSSION THREAD

**Discussion by jho2301** [woowacourse/tecoble#384] [General]
1. 리뷰어(코치) 매칭 어떻게 할지
    - BE는 브라운&구구 번갈아가면서? or 받고싶은 코치님한테 받기

    - FE는 포코 only
2. 블로그 개선팀 브리핑
3. 이름 컨벤션 다시 강조 & 기존의 이름들 바꾸기 & wiki에 고지하기
4. 지난 개선 사항 의견 중 반영할 것 다시 이야기해보기
    - 자기가 선정한 주제만 맡아서 하는 경향이 있다.
    - 특이사항이 없어도 회의는 주 1회가 좋을 것 같다.
    - 리뷰요청 슬랙봇 만들기
    - 카테고리 별로 볼 수 있게 하기
    - 제출에 기한 두기
5. 건의사항이 있는지 (회의, 글쓰기 리뷰 전반, 블로그 개선팀)
6. 다음 회의 언제 할지

의견 있으면 코멘트로 남겨주세요!

**Comment by hybeom0720** [comment]
+ 이전 글에 대해 깨진 링크 수정이나 댓글 질문 답변에 대한 의견 나눠보는 것도 좋을 것 같습니다.

**Comment by jho2301** [comment]
## 회의 정리


1. 리뷰어(코치) 매칭 어떻게 할지
    - BE는 브라운&구구 번갈아가면서? or 받고싶은 코치님한테 받기
    1. A/B그룹으로 나눠 번갈아가며 리뷰받기

3. 이름 컨벤션 다시 강조 & 기존의 이름들 바꾸기 & wiki에 고지하기
    1. 샐리가 기존 이름들 전체 변경
4. 지난 개선 사항 의견 중 반영할 것 다시 이야기해보기
    - 자기가 선정한 주제만 맡아서 하는 경향이 있다.
        - 현행 유지
    - 특이사항이 없어도 회의는 주 1회가 좋을 것 같다.
        - 매 주 진행하되, 특별한 안건이 없을 시 패스 (마스터가 금요일까지 고지)
    - 리뷰요청 슬랙봇 만들기
        - 블로그 개선팀 차후 개발일정에 리스트업
    - 카테고리 별로 볼 수 있게 하기
        - 개선팀이 진행
    - 제출에 기한 두기
        - 퀄리티가 낮아지지 않도록 다시 경계하자. 
        - 리뷰를 좀 더 꼼꼼히 하자
